### ISSUE TITLE
Add bcmath extension to pas-dev

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/princeton_ansible#3551]
Steps to recreate:

- Go to https://slavery-dev.princeton.edu/adminPrinceton
- Note that you get the error:

```
Can’t run Craft CMS :(

Your server doesn’t meet the following requirements to run Craft CMS:

    The [BCMath](https://php.net/manual/en/book.bc.php) extension is required.
```

**Comment by kevinreiss** [comment]
The craft upgrade guide also lists the intl extension as another new PHP requirement for v. 4. https://www.php.net/manual/en/book.intl.php.

**Comment by sandbergja** [comment]
Closed by #3552
